new release 0.10.0 (#81)
pkgcore/snakeoil: NEWS.rst

Release Notes
=============

snakeoil 0.10.0 (2022-09-18)
----------------------------

This release has various breaking changes, with various deprecated stuff

> removed. We are planing to remove more stuff in the future, so please be aware.

Reviewer: ```suggestion
removed. We are planing to remove more cruft in the future, so please be aware.
```

> All removed stuff will be listed in the release notes, and a better replacement

Reviewer: ```suggestion
All removals will be listed in the release notes, and a better replacement
```

> will be provided.

Reviewer: ```suggestion
will be provided. Please speak to us if they do not suit your needs.
```

This is also the first release to use ``flit`` for packaging, which simplifies
the build a lot. We include a makefile for convenience of running various build
commands.

> Also note that ``dist.distutils_extensions`` is now deprecated, and will be

Reviewer: I would explicitly include this as a bullet point as well, because then it's harder to miss.
